### PR TITLE
walt.id NFT Infrastructure - M1

### PR DESCRIPTION
**Pull request by philpotisk**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/1403

### CODE REVIEW

**Comment by philpotisk** [comment]
I see the author check is failing, which should be the case when the submitter of the deliverable is NOT the one who as submitted the application. I my case, however, it is failing although I am using the same GitHub account as you can see in the screenshot:

image

I hope somebody can give me a hint how to resolve this issue. thanks!

File: deliveries/walt-id_nft-infra-M1.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:**   [walt.id NFT Infrastructure](https://github.com/walt-id/Grants-Program/blob/master/applications/walt-id_nft-infra.md)
```
Comment (semuelle) [inline deliveries/walt-id_nft-infra-M1.md:5]:
```suggestion
* **Application Document:** https://github.com/walt-id/Grants-Program/blob/master/applications/walt-id_nft-infra.md
```

**Comment by 0xCaso** [comment]
@philpotisk any updates?

**Comment by philpotisk** [comment]
> @philpotisk any updates?

Hi @0xCaso thanks for reaching out and for clarifying the issues above. It is totally my bad, as I thought we need to submit the original planed 4 milestones one by one. But as we restructured the project that everything is bundled in a single milestone, we will just need to submit everything at once. Therefore it might make sense to delete this PR and to resubmit everything once the project is completed. right?

**Comment by 0xCaso** [comment]
In my opinion yes, if the whole project requires time and you didn't already finish it, I think it would be better to close this PR and open a new one when the milestone will be ready.

Otherwise, if you want to split the only milestone in more milestones, you can amend the grant opening a PR [here](https://github.com/w3f/Grants-Program) proposing a new schedule.

**Pull request closed without merging**